pages/overview.njk

```diff
@@ -0,0 +1,67 @@
+---
+layout: _templates/layout-basic.njk
+title: Overview
+order: 0
+---
+
+{% import '_templates/component/getstarted.njk' as getstarted %}
+
+{% call getstarted.section("Introduction to libraries") -%}
+
+  <p>We use several Adobe XD libraries to serve the users of the <strong>Red Hat Digital Design System</strong> and other web tools like FTS. Depending on your needs, you may request an invite to access one or more of our libraries. Each library is updated regularly so rest assured you will always be working with the latest colors, character styles, components, and more. To learn how to use libraries, watch the following video.</p>
```

Comment:
We should consider not using "FTS" as a term (since it will go away) and if we do, please use the full name.

Reply:
@dcaryll Eliminated the use of the FTS acronym where I could or replaced it with the long form.